src/services/api/activity-log/dto/activity.log.entry.interface.ts

```diff
@@ -50,7 +50,7 @@ import { IActivityLogEntryCalloutLinkCreated } from './activity.log.dto.entry.ca
     );
   },
 })
-export abstract class IActivityLogEntry {
+export class IActivityLogEntry {
```

Comment:
Why did you change that?

Reply:
Because the pagination requires an object that can be instantiated.
The same is done for Users, Organizations and all journeys.
I could extends the abstract class, but then the concrete class is included in the schema let me know if you can think of a solution.